4주차/[Lv.3]_등굣길/주다현.js

```diff
@@ -0,0 +1,17 @@
+//런타임 에러가 나는 코드랍니다... 아직 해결하지 못했어요ㅠㅠㅠㅠ
+
+function solution(m, n, puddles) {
+  const arr = Array.from(Array(n), () => Array(m).fill(1));
+  puddles.forEach((v) => {
+    const [x, y] = v;
+    arr[x - 1][y - 1] = 0;
+  });
+  for (let i = 1; i < n; i++) {
+    for (let j = 1; j < m; j++) {
```

Comment:
```suggestion
    for (let j = 1; j < m; j++) {
      if (arr[i][j] === null) {
        continue;
      }
```
`물에 잠긴 지역`이라면 다음으로 넘깁니다.